docs/client/containers.md

```diff
@@ -0,0 +1,80 @@
+# Containers
+
+## `Config`
+
+```python
+class Config(Container):
+    // temporary property to support simplified round robin block production in absence of randao & deposit mechanisms
+    num_validators: uint64
+```
+
+## `Checkpoint`
+
+```python
+class Checkpoint(Container):
+    root: Bytes32
+    slot: uint64
+```
+
+## `State`
+
+```python
+class State(Container):
+    config: Config
+    slot: uint64
```

Comment:
```suggestion
    slot: uint64
    latest_block_header: BlockHeader
```
this is needed to track and validate the block is being applied to the correct parent state, and is basic for chaining